README.md

```diff
@@ -123,7 +123,6 @@ When ran inside the CanDIG context, to properly implement authorization you'll
 have to do the following:
 
 1. Make sure the CHORD_PERMISSIONS is set to "false"
-2. Set INSIDE_CANDIG to "true"
 3. Provide the URL for the OPA instance in CANDIG_OPA_URL
```

Comment:
It shouldn't matter in the rendering, but you'll probably want to change this "3" to 2"

Reply:
Ah yes nice catch, just changed it to 2.